project.clj

```diff
@@ -89,4 +89,8 @@
                               [:name "Ryan Brush"]
                               [:url "http://www.clara-rules.org"]]]
   :deploy-repositories [["snapshots" {:url "https://oss.sonatype.org/content/repositories/snapshots/"
-                                      :creds :gpg}]])
+                                      :creds :gpg}]
+                        ["clojars" {:url "https://repo.clojars.org"
+                                    :username :env/clojars_username
```

Comment:
From some quick googling it looks like we'll need these variables to be added using the instructions at https://docs.travis-ci.com/user/environment-variables/#encrypting-environment-variables.  Is that what you're thinking?  I could have missed something of course, I haven't looked at the travis docs in much depth to be honest.

Reply:
@WilliamParker yes, I have already added the variables from the build configuration in Travis CI console